Application/global_settings_manager.py

```diff
@@ -68,6 +68,17 @@ def load_global_settings(self):
         settings = QSettings()
 
         settings.beginGroup("global-settings")
+
+        settings.beginGroup("user-settings")  # open user settings
+
+        # Sets the padding and the max width to global settings entity
+        self.global_settings_entity.table_padding = settings.value("table_padding")
+        self.global_settings_entity.table_maximum_width = settings.value("table_maximum_width")
+
+        # Sets the cell size with width, index 0, and height, index 1 to global settings entity
+        self.global_settings_entity.table_cell_size.append(settings.value("table_cell_size_width"))
+        self.global_settings_entity.table_cell_size.append(settings.value("table_cell_size_height"))
+
```

Comment:
After this line, we need to end the group "user-settings". By not removing it, we are encapsulating the "encoding-buttons" into the "user-settings" group. This changes the hierarchy and will interfere with the encoding button persistence.